extension/extension.js

```diff
@@ -1,6 +1,10 @@
 const axios = require('axios');
 const Channel = require('jschannel');
 
+const instance = axios.create({
+  validateStatus: () => true
```

Comment:
When you set the `validateStatus` to `null` you avoid calling the `validateStatus` function at all and the result is the same.

Reply:
You are right.

I used the same format that the other places used for consistency - https://github.com/apiaryio/console-proxy/blob/master/src/JsAgent/JsAgent.js#L6 and https://github.com/apiaryio/console-proxy/blob/master/public/apiary-proxy.js#L2

Reply:
I would rather change on the other places as well ;)

Reply:
I find having the validator function returning true being more explicit and clear then having `validateStatus: null` there.

Anyway I am going to create a PR with that change you guys can vote using the review. :)

Reply:
I just realized it was already deployed. Well, I think anyway it's a minor optimization, not worth the change IMHO ;)